### DISCUSSION TITLE
Craft 5

### DISCUSSION THREAD

**Discussion by tyssen** [solspace/craft-express-forms#125] [Questions & Answers] [answered]
Are there plans to make a Craft 5 version available?

**Comment by kjmartens** [comment, marked as answer]
Hi @tyssen,

Unfortunately, Express Forms has now been discontinued. You can read more about it here:
[Important Notice - Express Forms Discontinued](https://github.com/solspace/craft-express-forms/issues/126)

We apologize for any inconvenience caused.
